server/resource/middleware/create.js

```diff
@@ -56,7 +56,10 @@ module.exports = async function(req, res) {
 
   log.info({query, definition: this.definition, reqId: req.id}, 'Creating a resource');
 
-  const result = await this.db.one(query).catch(r => r);
+  const result = await this.db.tx(t => {
+    const primaryTableQuery = t.one(query);
+    return t.batch([primaryTableQuery]);
+  }).catch(r => r);
 
   if (_.isError(result)) {
```

Comment:
I'll likely need to check if `result` is an error, or an array of errors. Or just switch to good old try/catch. But for now, this passes the integration tests, so I'm going with it. I'll revisit later.